app/sidekiq/simple_forms_api/notification/send_notification_email_job.rb

```diff
@@ -60,6 +59,17 @@ def time_to_send
       def statsd_tags
         { 'service' => 'veteran-facing-forms', 'function' => "#{form_number} form submission to Lighthouse" }
       end
+
+      def handle_exception(e)
+        Rails.logger.error(
+          'Error sending simple forms notification email',
+          message: e.message,
```

Comment:
Want to confirm  there's no possibility for PII / sensitive information being leaked in this log.

Reply:
Nope this should be good for PII. The error message would not include PII unless it somehow accesses the form data from inside the form submission on its own.